iib/workers/s3_utils.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+
+import boto3
+
+from iib.exceptions import IIBError
+from iib.workers.config import get_worker_config
+
+log = logging.getLogger(__name__)
+
+conf = get_worker_config()
+
+
+def upload_file_to_s3_bucket(file_path, key_prefix, file_key):
+    """
+    Upload artifact file to AWS S3 bucket.
+
+    :param str file_path: the path of the file locally where the artifact file is stored
+    :param str key_prefix: the logical location of the file in the S3 bucket
+    :param str key_prefix: the name of the file in S3 bucket
```

Comment:
`key_prefix` 2 times? I believe it's a typo, the last one should be `file_key` IMO